All snd_emu_chip_details instances are statically allocated and all users of
this new member are within one module.

diff --git a/include/sound/emu10k1.h b/include/sound/emu10k1.h
index 10ee28e..1515459 100644
--- a/include/sound/emu10k1.h
+++ b/include/sound/emu10k1.h
@@ -1670,10 +1670,12 @@ struct snd_emu_chip_details {
 	unsigned char spi_dac;      /* SPI interface for DAC */
 	unsigned char i2c_adc;      /* I2C interface for ADC */
 	unsigned char adc_1361t;    /* Use Philips 1361T ADC */
-	unsigned char invert_shared_spdif; /* analog/digital switch inverted */
 	const char *driver;
 	const char *name;
 	const char *id;		/* for backward compatibility - can be NULL if not needed */
+#ifndef __GENKSYMS__
+	unsigned char invert_shared_spdif; /* analog/digital switch inverted */
+#endif
 };
 
 struct snd_emu1010 {
